public/content/staking/solo/index.md

```diff
@@ -13,31 +13,31 @@ summaryPoints:
+**Home staking increases the decentralization of the Ethereum network**, making Ethereum more censorship-resistant and robust against attacks. Other staking methods may not help the network in the same ways. Home staking is the best staking option for securing Ethereum.
 
 An Ethereum node consists of both an execution layer (EL) client, as well as a consensus layer (CL) client. These clients are software that work together, along with a valid set of signing keys, to verify transactions and blocks, attest to the correct head of the chain, aggregate attestations, and propose blocks.
 
-Solo stakers are responsible for operating the hardware needed to run these clients. It is highly recommended to use a dedicated machine for this that you operate from home–this is extremely beneficial to the health of the network.
+Home stakers are responsible for operating the hardware needed to run these clients. It is highly recommended to use a dedicated machine for this that you operate from home–this is extremely beneficial to the health of the network.
 
-A solo staker receives rewards directly from the protocol for keeping their validator properly functioning and online.
+A home staker receives rewards directly from the protocol for keeping their validator properly functioning and online.
 
-## Why stake solo? {#why-stake-solo}
+## Why stake home? {#why-stake-solo}
 
-Solo staking comes with more responsibility but provides you with maximum control over your funds and staking setup.
+Home staking comes with more responsibility but provides you with maximum control over your funds and staking setup.
 
 <CardGrid>
   <Card title="Earn fresh ETH" emoji="💸" description="Earn ETH-denominated rewards directly from the protocol when your validator is online, without any middlemen taking a cut." />
   <Card title="Full control" emoji="🎛️" description="Keep your own keys. Choose the combination of clients and hardware that allows you to minimize your risk and best contribute to the health and security of the network. Third-party staking services make these decisions for you, and they don't always make the safest choices." />
-  <Card title="Network security" emoji="🔐" description="Solo staking is the most impactful way to stake. By running a validator on your own hardware at home, you strengthen the robustness, decentralization, and security of the Ethereum protocol." />
+  <Card title="Network security" emoji="🔐" description="Home staking is the most impactful way to stake. By running a validator on your own hardware at home, you strengthen the robustness, decentralization, and security of the Ethereum protocol." />
 </CardGrid>
 
-## Considerations before staking solo {#considerations-before-staking-solo}
+## Considerations before staking home {#considerations-before-staking-solo}
```

Comment:
I think this would read better if it was "Considerations before home staking", rather than "staking home"

Reply:
Yes, I agree. Updated✅